config.yaml

```diff
@@ -4,12 +4,23 @@
 # openstackrc > token > password
```

Comment:
Yeah since config.yaml has been deprecated for a while now we should port it to config schema.

Here is an example of a nested config - https://github.com/StackStorm-Exchange/stackstorm-vsphere/blob/master/config.schema.yaml